coreax/kernel.py

```diff
@@ -335,9 +335,15 @@ def divergence_x_grad_y_elementwise(self, x: ArrayLike, y: ArrayLike) -> Array:
 
 
 class SquaredExponentialKernel(Kernel):
-    """
+    r"""
     Define a squared exponential kernel.
 
+    The squared exponential kernel is defined as
+    :math:`k: \mathbb{R}^d\times \mathbb{R}^d \to \mathbb{R}`,
+    :math:`k(x, y) = \text{output_scale} \exp(\frac{||x-y||^2}
```

Comment:
In CONTRIBUTING.md we request that mathematical expressions aren't split across multiple lines (hence the preference for using symbols)